configurable-rules/string-schemas-length-defined/README.md

```diff
@@ -0,0 +1,78 @@
+# String schemas length defined
+
+Authors:
+- [`adamaltman`](https://github.com/adamaltman) Adam Altman (Redocly)
+ 
+## What this does and why
+
+This requires `minLength` and `maxLength` properties set on a `string` where `enum` isn't defined.
+
+## Code
+
+The rule checks that a `string` uses the `minLength` and `maxLength` keywords unless an `enum` is defined.
+
+It does this by using a combination `requireAny` and `mutuallyRequired`.
+
+```yaml
+  rule/string-schemas-length-defined:
+    subject: 
+      type: Schema
+    where: 
+      - subject: 
+          type: Schema
+          property: type
+        assertions:  
+          const: string
+    assertions: 
+      requireAny: 
+        - minLength
+        - maxLength
+        - enum
+      mutuallyRequired: 
+        - minLength
+        - maxLength
```

Comment:
Albeit it's correct, the opposite order seems more natural to me. I think it's easier to read if the top-level `subject` and `assertions` are grouped together and the `where` lies above or or below. What do you think?